.github/workflows/hydroshare-pull-requests.yml

```diff
@@ -32,10 +32,6 @@ jobs:
       run: |
         printf "%s\n" c | ./local-dev-first-start-only.sh
 
-    - name: Start HydroShare
-      run: |
-        docker-compose -f local-dev.yml up -d
-        sleep 60
```

Comment:
Is this not needed? The last line of local-dev-first-start-only is `docker-compose -f local-dev.yml down` so HS isn't running.